test/e2e/ingress/multi_path_backend.go

```diff
@@ -32,7 +32,7 @@ type BackendConfig struct {
 
 type MultiPathIngressConfig struct {
 	GroupName  string
-	GroupOrder int32
+	GroupOrder int64
```

Comment:
any reason for this change?

Reply:
Because it's an int64 [here](https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/9935d4847f11e2ac07d50e734e4c5bb770f896b5/pkg/ingress/group_loader.go#L263). I figured they should be the same type, even if changing this line doesn't effect the test passage either way right now.

Reply:
cool, thanks